docs/release/trg-0/trg-1-06.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TRG 1.06 - Administrator's Guide (User Assistance)
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 31-Jan-2024  | created initial Draft                  |
+
+## Why
+
+Administrator's Guide is intended for administrators that install and operate the solution.
+
+## Description
+
+- The Admin Guide **has to** be stored in the folder: ```docs/admin/```
+- Admin Guide is available with the software at the same time.
+- Admin Guide is correct and up to date; English is a must.
+- The documentation is of appropriate maturity to be handed over to any Operations Company with global business practice intentions.
```

Comment:
What is "appropriate maturity"
As open source project we do not hand over anything. We provide it and invite everyone to use on their own risk